### ISSUE TITLE
how do I get the coordinates?

### ISSUE THREAD

**Issue by savageduckerino** [maxenxe/HQ-Trivia-Bot#44]
I've tried with multiple programs, I put in the coordinates, then it takes screenshot of somewhere totally else on my screen...

**Comment by kiikoh** [comment]
i have noticed this also, i found the the first two parameters are for the image size, and the last two are positional but I cant figure out how it works

**Comment by maxenxe** [comment]
0;0 are the top left coordinates of your screen, so the command work like this .crop('width in pixel','height in pixel', 'horizontal pixels from the left of your screen', 'vertical pixel from the top of your screen')

**Comment by maxenxe** [comment]
https://www.youtube.com/watch?v=QzdqT0HTGLs this vid shows how to set the bot up

**Comment by Frosty704** [comment]
yeah its kinda confusing but i got it now messing around with the coords lol

**Comment by U-r-Unknown** [comment]
I have working bot which works on all trivia games. No worries about ban as this is PC software. No need of any programming knowledge. If interested mail me onlineuse321@gmail.com. (setting up this is frustrating and it gives answer after time out)
